lib/ansible/plugins/connection/winrm.py

```diff
@@ -49,14 +54,14 @@
 from ansible.utils.hashing import secure_hash
 from ansible.utils.path import makedirs_safe
 from ansible.utils.unicode import to_bytes, to_unicode, to_str
-from ansible.utils.vars import combine_vars
 
 try:
     from __main__ import display
 except ImportError:
     from ansible.utils.display import Display
     display = Display()
 
+HALF_MB = 2**19  # 0.5MB chunks
```

Comment:
Better to name this after what it is used for.  BUFSIZE is used in other connection plugins.